index.bs

```diff
@@ -5679,39 +5674,34 @@ steps may throw an exception.
       <dt>If IsArray(|input|)</dt>
       <dd>
 
-        1. Let |len| be the ToLength(Get(|input|,
+        1. Let |len| be [=!=] ToLength(Get(|input|,
```

Comment:
Fun. I think implementations are likely to bypass the getter here for exotic array objects and access the internal length directly (Chrome does!).

Switched these to ? for the cases where the input is not a clone for now. 

(I local testing I don't see the length override getting called even when I invoke length on an instance. Hrm.)